docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -50,15 +50,15 @@ const StyledAppContainer = styled(AppContainer, {
           marginBottom: 198,
         },
         '&& .description.ad': {
-          marginBottom: 40,
+          marginBottom: 0,
         },
       }),
       ...(hasTabs && {
         '&& .component-tabs .MuiTabs-root': {
-          marginBottom: 193,
+          marginBottom: 198,
```

Comment:
Fix https://github.com/mui/material-ui/pull/35938#discussion_r1155108624

There is still a layout shift with Joy UI, but it's because the theme are not correctly isolated, the Ad renders with Joy UI theme, not with MUI's branding theme 🙃